app/src/main/java/com/capstone/event_finder/network/EventApi.java

```diff
@@ -17,19 +17,19 @@
 import retrofit2.Callback;
 import retrofit2.Response;
 
-public class EventApi {
+public class EventApi implements GetAPIEventsHandler {
```

Comment:
🛠 The API itself theoretically does not need to implement the interface.  It's just needed for the callback parameter in `getAPIEvents` below.